packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -88,9 +88,9 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsETH = true;
   readonly _supportsETHInfo = true;
   readonly _supportsEthSwitchChain = false;
```

Comment:
Apparently supported, though I didn't get this to work using Tally with other dApps
https://github.com/tahowallet/extension/blob/d5811da18605927dda92280092f4d41f3e432d58/background/services/internal-ethereum-provider/index.ts#L289-L297

Will give this some more testing and PR the capability update in a follow-up PR after confirming this actually does what it says on the box